src/main/java/RandomProvider.java

```diff
@@ -0,0 +1,3 @@
+public interface RandomProvider {
+    int getRandom();
```

Comment:
int getRandom(int rangeTo);
int getRandom(int rangeFrom, int rangeTo);

We want to set a range for random numbers from 0 to bound (exclusive).